### PR TITLE
Update version and add autoupdate strategy

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
## Problem

Version 1.4.0 has been released. https://github.com/borgbackup/borg/releases/tag/1.4.0 Also, in `install_borg_with_pip` the version is retrieved with `ynh_app_upstream_version` although there was no upstream version defined (if I understand it correctly)? See https://github.com/YunoHost-Apps/borg_ynh/blob/master/scripts/_common.sh#L17

## Solution

So, I updated the version. I also added the borg source from GitHub to have an autoupdate strategy. As borg is still installed with pip I set `prefetch=false`.

Edit: Also added some missing APT sources from https://borgbackup.readthedocs.io/en/stable/installation.html#debian-ubuntu

- libzstd-devel
- xxhash-devel

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by CodeShakingSheep** [comment]
!testme

**Comment by CodeShakingSheep** [comment]
!testme

**Comment by CodeShakingSheep** [comment]
!testme

**Comment by CodeShakingSheep** [comment]
!testme

**Comment by CodeShakingSheep** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ install_borg_with_pip () {
     venvpy="$install_dir/venv/bin/python3"
 
     ynh_exec_as "$app" "$venvpy" -m pip install --upgrade setuptools wheel
-
+
```
Comment (kay0u) [inline scripts/_common.sh:16]:
```suggestion

```

**Comment by kay0u** [comment]
Thank for your work, I kept the current implementation of the autoupdate in the manifest.toml as we can see [it works well](https://github.com/YunoHost-Apps/borg_ynh/pull/173), but

**Pull request merged**